kernelci.org/content/en/docs/org/tsc/votes.md

```diff
@@ -9,14 +9,49 @@ outcome and a more detailed section listing each individual vote.
 
 ## Summary
 
-| Date                      | Motion                               | Status   |
-|---------------------------|--------------------------------------|----------|
-| [2022-06-09](#2022-06-09) | Add TSC member: Denys                | Approved |
-| [2022-02-10](#2022-02-10) | Remvove TSC member: Lakshmipathi     | Approved |
-| [2021-07-13](#2021-07-13) | Add TSC member: Alice                | Approved |
+| Date                      | Motion                                                                  | Status   |
+|---------------------------|-------------------------------------------------------------------------|----------|
+| [2022-06-09](#2022-06-09) | Add TSC member: Denys                                                   | Approved |
+| [2022-02-10](#2022-02-10) | Remvove TSC member: Lakshmipathi                                        | Approved |
+| [2021-07-13](#2021-07-13) | Add TSC member: Alice                                                   | Approved |
+| [2021-12-08](#2021-12-08) | Motion: Adopt the currently documented TSC rules as a starting point    | Approved |
+| [2021-12-08](#2021-12-08) | Motion: Add mgalka to the GitHub admins team                            | Approved |
+| [2021-12-08](#2021-12-08) | Motion: Confirm a list of the components to which each member of the ...| Approved |
```

Comment:
This summary is not very helpful.  Something like "List of SysAdmin resources" would be better.

Also please drop `Motion:` as it's already in the Motion column.

Reply:
One more thing, the year should be 2022 and not 2021.  Then the motions should be at the top of the table to have the latest ones first, at least that's how I started this table.  I think it's more convenient this way.